--- a/linux_lib/libcrystalhd/libcrystalhd_priv.h
+++ b/linux_lib/libcrystalhd/libcrystalhd_priv.h
@@ -62,10 +62,10 @@ enum _crystalhd_ldil_globals {
 	BC_INPUT_MDATA_POOL_SZ_COLLECT  = 256,		/* Input Meta Data Pool size for collector */
 	BC_MAX_SW_VOUT_BUFFS    = BC_RX_LIST_CNT,	/* MAX - pre allocated buffers..*/
 	RX_START_DELIVERY_THRESHOLD = 0,
-	PAUSE_DECODER_THRESHOLD = 12,
-	RESUME_DECODER_THRESHOLD = 5,
-	FLEA_RT_PD_THRESHOLD = 14,
-	FLEA_RT_PU_THRESHOLD = 3,
+	PAUSE_DECODER_THRESHOLD = 6,
+	RESUME_DECODER_THRESHOLD = 3,
+	FLEA_RT_PD_THRESHOLD = 7,
+	FLEA_RT_PU_THRESHOLD = 2,
 	HARDWARE_INIT_RETRY_CNT = 10,
 	HARDWARE_INIT_RETRY_LINK_CNT = 1,
 };
